boards/arm/sam34/sam4cmp-db/scripts/Make.defs

```diff
@@ -32,8 +32,8 @@ ifneq ($(CONFIG_DEBUG_NOOPT),y)
   ARCHOPTIMIZATION += $(MAXOPTIMIZATION) -fno-strict-aliasing
 endif
 
-ARCHCFLAGS = -fno-builtin
-ARCHCXXFLAGS = -fno-builtin -fno-exceptions -fcheck-new
+ARCHCFLAGS = -fno-common -fno-builtin
+ARCHCXXFLAGS = -fno-common -fno-builtin -fno-exceptions -fcheck-new
```

Comment:
Note: strange that `boards/arm/sam34/sam3u-ek/scripts/Make.defs` has `-fno-rtti` and here we do not have it. The same in other files. This is not related to this PR, but notice in general. Maybe we should have a separate PR to unify `-fno-rtti` usage

Reply:
Agree.